### ISSUE TITLE
Rename Widgets::CollectionView fields.

### ISSUE THREAD

**Issue by caspervonb** [ruhoh/ruhoh.rb#158]
@plusjade considering every other collection views have migrated away from an underscore convention, would it not be appropriate for widgets to do the same?

So it would be this.path, this.config rather than this_path and this_config.

**Comment by plusjade** [comment]
@caspervonb yes you are right. When I re-implemented the widgets I was in a "get this damned thing working". Since then I haven't really cleaned up the widgets implementation.  I agree, we should pass in an object that responds to the arbitrary configuration paramaters. Please feel free to submit a refactored pull request. I'll keep this open so I'll get to it eventually if you can't. thanks!

**Comment by caspervonb** [comment]
I'll try to get to it this weekend.
